vpr/src/pack/re_cluster_util.cpp

```diff
@@ -340,10 +404,10 @@ static void fix_cluster_pins_after_moving(const ClusterBlockId clb_index) {
         }
     }
 
-    for (j = 0; j < num_output_ports; j++) {
-        ClusterPortId output_port_id = cluster_ctx.clb_nlist.find_port(clb_index, block_type->pb_type->ports[num_input_ports + j].name);
-        for (k = 0; k < pb->pb_graph_node->num_output_pins[j]; k++) {
-            pb_graph_pin = &pb->pb_graph_node->output_pins[j][k];
+    for (iport = 0; iport < num_output_ports; iport++) {
```

Comment:
Good to comment what each block of high level code (e.g. this for loop) is doing if you can.